include/mgba-util/socket.h

```diff
@@ -438,6 +441,56 @@ static inline int SocketPoll(size_t nSockets, Socket* reads, Socket* writes, Soc
 	return result;
 }
 
+static inline int SocketResolveHost(const char* addrString, struct Address* destAddress) {
+	int result = 0;
+#ifdef USE_GETHOSTBYNAME
+	// TODO: This isn't threadsafe. Fix!
```

Comment:
Perhaps make this a #warning, especially in the #else case.

Reply:
Why would there be a warning in the #else case?

Reply:
To denote that that case is strongly recommended against. It shouldn't be hit in any existing compilation path atm, so it'd be a good idea to tell people _don't hit this path_ if at all possible.